chat-plugins/info.js

```diff
@@ -328,7 +328,7 @@ var commands = exports.commands = {
 	dt: 'details',
 	details: function (target) {
 		if (!target) return this.parse('/help details');
-		CommandParser.commands.data.apply(this, arguments);
+		this.run('data');
```

Comment:
The original ``commands`` is the one locally defined at ``info.js``.
After the patch, it becomes the one locally defined at ``commands.js`` (``CommandParser.commands``). ``/details`` will keep its old reference and run an outdated ``/data`` command.